opencadc.gradle

```diff
@@ -18,31 +18,32 @@ checkstyle {
 
 sourceSets {
     test {
-        resources.srcDirs += 'src/test/resources'
+        //resources.srcDirs += 'src/test/resources'
     }
```

Comment:
Having the `src/test/resources` injected is already implied.  This `test` block can be removed.